libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -97,7 +97,7 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
     this._contentClass = value;
     if (isReactNode(this.reactNodeRef.nativeElement)) {
       this.reactNodeRef.nativeElement.setProperty('className', classnames(value));
-      this.changeDetectorRef.detectChanges();
+      this.markForCheck();
```

Comment:
Did you encounter any issues that warrant this change? `markForCheck()` does more than `this.changeDetectorRef.detectChanges()`.

Reply:
When first using the Calendar, I noticed that if it shared a template with another React-wrapped component, and that other component had any attributes that would later become property-bindings, that the Calendar would fail, having not correctly bound its own properties. I managed to work out that this was somehow related to the renderer. Changing this fixed the problem.